The raw YAML output fields are sometimes printed in non-alphabetical 
order, causing the raw text comparison to sometimes fail.  The parsed 
output is subsequently compared, and that should be all that matters.

diff --git a/tests/unit/output/test_yaml_output.py b/tests/unit/output/test_yaml_output.py
index b5e3a0a..88132f7 100644
--- a/tests/unit/output/test_yaml_output.py
+++ b/tests/unit/output/test_yaml_output.py
@@ -62,7 +62,6 @@ class TestYAMLOutput(BaseAWSCommandParamsTest):
             "  UserId: EXAMPLEUSERID\n"
             "  UserName: testuser-51\n"
         )
-        self.assertEqual(stdout, expected)
         parsed_output = self.yaml.load(stdout)
         self.assertEqual(self.parsed_response, parsed_output)
 
